src/InsetOrderOptimizer.cpp

```diff
@@ -164,7 +164,8 @@ size_t InsetOrderOptimizer::getOuterRegionId(const VariableWidthPaths& toolpaths
     return outer_region_id;
 }
 
-BinJunctions InsetOrderOptimizer::variableWidthPathToBinJunctions(const VariableWidthPaths& toolpaths, const bool pack_regions_by_inset, const bool center_last, std::set<size_t>* p_bins_with_index_zero_insets)
+BinJunctions InsetOrderOptimizer::variableWidthPathToBinJunctions(const VariableWidthPaths& toolpaths, const bool pack_regions_by_inset,
```

Comment:
I'm not going to change it (since frankly it isn't _that_ important and we're inconsistant with it as well), but in C++ I think our code style is that if it's too long for a single line, then:

~~~
return-type className::functionName
(
    type parameter0,
    type parameter2,
    ....
)
~~~

Reply:
https://github.com/Ultimaker/CuraEngine/pull/1545/commits/f90088a316b15e675230ced08344dd2426262b09